### PR TITLE
Added Get and Post for Account

### PR DESCRIPTION
**Pull request by yantaozhu**
# Description
Created the methods in the `AccountController` and `AccountService` for GET and POST requests.

### Changes
* Added `getAllAccounts` method
* Added `getAccountByAccountId` method
* Added `getAccountByUserId` method
* Added `insertAccount` method

### Test the Get Method
Put `http://localhost:5000/p1/accounts/` in the URL to get all accounts
Put `http://localhost:5000/p1/accounts/id/{accountId}` in the URL to get an account with a specific id
Put `http://localhost:5000/p1/accounts/user/{userId}` in the URL to get all accounts belong to the user with that user id

### Test the Post Method
Put `http://localhost:5000/p1/accounts/new/{userId}` in the URL

The body should be raw -> JSON
```
{
    "accountBalance" : "1000"
}
```

### CODE REVIEW

File: src/main/java/com/revature/controllers/AccountController.java [hunk 1]
```diff
@@ -16,4 +20,40 @@ public class AccountController {
...
+        try {
+            return ResponseEntity.accepted().body(accountService.getAccountByUserId(userId));
+        } catch (IllegalArgumentException e) {
+            e.printStackTrace();
+            return ResponseEntity.badRequest().body(e.getMessage());
+        }
+    }
+
+    @PostMapping("/new/{userId}")
+    public ResponseEntity<Object> insertLoan(@RequestBody Account account, @PathVariable("userId") int userId) {
```
Comment (CalebMcOlin) [inline src/main/java/com/revature/controllers/AccountController.java]: I believe this method should be called `insertAccount` not `insertLoan`

File: src/main/java/com/revature/models/Account.java [hunk 2]
```diff
@@ -18,20 +18,20 @@ public class Account {
 
     @ManyToOne(fetch = FetchType.EAGER, cascade = CascadeType.ALL) // Creating the FK
     @JoinColumn(name = "userId")
-    private User userId;
+    private User user;
```
Comment (CalebMcOlin) [inline src/main/java/com/revature/models/Account.java:21]: Good catch!! I missed that when I was refactoring during my last PR.

**Review by CalebMcOlin — CHANGES_REQUESTED** [review]
Very good job. All testing passed on my end.

The only thing is, we should rename the function in the `AccountController.java` to `insertAccount`

File: src/main/java/com/revature/controllers/AccountController.java (hunk 1 above)
Comment (yantaozhu) [reply src/main/java/com/revature/controllers/AccountController.java]: Good catch, I was taking some of your code to change into the method for the account section and I totally didn't catch that.

**Review by CalebMcOlin — APPROVED** [review]
Looks great!

**Review by brytat — APPROVED** [review]

**Pull request merged**